jobbergate-api/jobbergate_api/apps/job_submissions/routers.py

```diff
@@ -104,7 +104,7 @@ async def job_submission_create(
 
     logger.debug(f"Job-submission created: {job_submission_data=}")
 
-    return job_submission_data
+    return JobSubmissionResponse(**job_submission_data)
```

Comment:
This line needs more attention. Returning just `job_submission_data`, as in the original code, results in a Pydantic validation error, which is just weird because we can create the model in the new code with no problems. 

Mypy is complaining about the new code since the argument after ** must be a mapping.

Reply:
solved in [60cea43](https://github.com/omnivector-solutions/jobbergate/pull/189/commits/60cea43c3b15c7a83086f7bd5e4683e8915f3359)